frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+
+
+export default function Footer() {
+  return (
+    <footer className="border-t bg-slate-200 dark:bg-slate-800">
+      <div className="container px-4 py-4 md:py-8 text-slate-800 dark:text-slate-200">
+        <div className="grid gap-8 sm:grid-cols-2 md:grid-cols-4">
+          <div className="space-y-4">
+            <h3 className="text-lg font-semibold">About OWASP</h3>
+            <ul className="space-y-2 text-sm">
+              <li>
+                <a className="text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-100" href="#">
+                  Our Mission
+                </a>
+              </li>
+              <li>
```

Comment:
All of these `<li>` elements seem identical except for the `href` value and actual tab text. 
I think it would be nicer to have it rendered by using `.map()`and going over pre-defined array of objects with these values. 🤔 Something like this:

```
  <ul>
    {tabs.map((tab) => (
      <li>
        <a key={tab.id} className="text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-100" href={tab.link}>
          {tab.name}
        </a>
      </li>
    ))}
  </ul>
  ```

Reply:
now i have used map